www/metadata.php

```diff
@@ -5,6 +5,12 @@
 
 require_once('../www/_include.php');
 
+use SAML2\Constants;
+use SimpleSAML\Utils\Auth as Auth;
```

Comment:
Not that it changes much, by the `as _____` isn't necessary when the text you provide after `as` exactly matches the class name. In other words, this...
```php
use SimpleSAML\Utils\Auth as Auth;
```
... is the same as this:
```php
use SimpleSAML\Utils\Auth;
```